src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneStoppedNode.kt

```diff
@@ -97,11 +98,16 @@ internal data class StandaloneStoppedNode(
         return ssh.execute("cat $unpackedProduct/work/catalina.pid").output.trim().toInt()
     }
 
+    /**
+     * Possible side effect: when wrong password provided increases cwd_user.invalidPasswordAttempts and that might result in
```

Comment:
Can this happen via this automation for some reason?

Reply:
What do you mean by automation? Comment explains it does happen when using rest calls with wrong password provided

Reply:
From what I understand the described behavior is related to the Jira produce and I understand that the problem around captcha can happen when an automation is using the wrong password. 

My question is: Can this wrong password happen in our solution? If yes than maybe we could detect in the code and inform with verbose exception message.

Reply:
I think this scenario is 99% unlikely to appear again. We control password being set in db and used for virtual user.
I've found no other cases of hardcoded usage of credentials.
And for that 1% of uncertanity there is a comment, detecting counter of invalidPasswordAttempts before login would be complicated to implement

Reply:
This is a good source code comment. In case where you use a wrong password and then somehow reuse that node with a correct password, VUs could fail due to captcha. I dunno how to construct such a use case, but apparently it already has bitten @mgrzaslewicz so documenting this "gotcha" is a plus.